nsqd/context.go

```diff
@@ -3,3 +3,8 @@ package nsqd
 type context struct {
 	nsqd *NSQD
 }
+
+type notifyContext struct {
+	notifyType int
+	v          interface{}
+}
```

Comment:
this type definition probably belongs in `nsqd/lookup.go` (where it's used)

Reply:
done.